prow/cmd/tackle/main.go

```diff
@@ -33,6 +33,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	v1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
Please follow the pattern: extensionsv1beta1

Reply:
Fixed